code/modules/client/preference_setup/traits/trait_defines.dm

```diff
@@ -0,0 +1,212 @@
+		return "Full Body Prosthetics cannot bleed."
+	// If a species lacking blood is added, it is suggested to add a check for them here.
+	return ..()
+
+/datum/trait/modifier/physical/weak
+	name = "Weak"
+	desc = "A lack of physical strength causes a diminshed capability in close quarters combat."
+	modifier_type = /datum/modifier/trait/weak
+
+
+/datum/trait/modifier/physical/inaccurate
+	name = "Inaccurate"
+	desc = "You're rather inexperienced with guns, you've never used one in your life, or you're just really rusty.  \
+	Regardless, you find it quite difficult to land shots where you wanted them to go."
+	modifier_type = /datum/modifier/trait/inaccurate
+
+
+// These two traits might be borderline, feel free to remove if they get abused.
+/datum/trait/modifier/physical/high_metabolism
+	name = "High Metabolism"
+	modifier_type = /datum/modifier/trait/high_metabolism
+
+/datum/trait/modifier/physical/high_metabolism/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics do not have a metabolism."
+	return ..()
+
+
+/datum/trait/modifier/physical/low_metabolism
+	name = "Low Metabolism"
+	modifier_type = /datum/modifier/trait/low_metabolism
+
+/datum/trait/modifier/physical/low_metabolism/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics do not have a metabolism."
+	return ..()
+
+
+/datum/trait/modifier/physical/cloned
+	name = "Cloned"
+	desc = "At some point in your life, you died and were cloned."
+	modifier_type = /datum/modifier/cloned
+
+/datum/trait/modifier/physical/cloned/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics cannot be cloned."
+	return ..()
+
+
+/datum/trait/modifier/physical/no_clone
+	name = "Cloning Incompatability"
+	modifier_type = /datum/modifier/no_clone
+
+/datum/trait/modifier/physical/no_clone/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics cannot be cloned anyways."
+	return ..()
+
+
+/datum/trait/modifier/physical/no_borg
+	name = "Cybernetic Incompatability"
+	modifier_type = /datum/modifier/no_borg
+
+/datum/trait/modifier/physical/no_borg/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics are already partly or fully mechanical."
+	return ..()
+
+
+
+// 'Mental' traits are just those that only sapients can have, for now, and generally involves fears.
+// So far, all of them are just for fluff/don't have mechanical effects.
+/datum/trait/modifier/mental
+	category = "Mental"
+
+/datum/trait/modifier/mental/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		if(setup.get_FBP_type() == PREF_FBP_SOFTWARE)
+			return "Drone Intelligences cannot feel emotions."
```

Comment:
IDK about adding that given those pesky pesky A-classes, but it might be appropriate the whole "can't feel emotions but can fake them pretty well" thing they've got going on.

(Also if we add racism mental traits I don't want to deprive people of their racism bots)

Reply:
I'm not sure I agree with this point.